process/block/preprocess/miniBlocksCompaction.go

```diff
@@ -369,5 +370,5 @@ func (mbc *miniBlocksCompaction) removeTxHashesFromMiniBlock(miniBlock *block.Mi
 		preservedTxHashes = append(preservedTxHashes, txHash)
```

Comment:
using indexes instead of append is faster.

Reply:
same, is out of order insertion